meshnets/modules/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Init"""
+from .decoder import *
```

Comment:
Should we be more explicit?

Reply:
Only if there is something that we specifically do not want to be importable (I think)

Reply:
This isn't a good practice, since it omits what is being imported. Import all submodules you want instead.